generated/cost-management/microsoft.costmanagement/2019-03-01-preview/types.md

```diff
@@ -10,8 +10,18 @@
 * **properties**: [ConnectorProperties](#connectorproperties): Connector properties
 * **type**: 'Microsoft.CostManagement/cloudConnectors' (ReadOnly, DeployTimeConstant): The resource type
 
+## Resource Microsoft.CostManagement/externalBillingAccounts@2019-03-01-preview (ReadOnly)
+* **Valid Scope(s)**: Tenant
+### Properties
+* **apiVersion**: '2019-03-01-preview' (ReadOnly, DeployTimeConstant): The resource api version
+* **id**: string (ReadOnly, DeployTimeConstant): The resource id
+* **kind**: string (ReadOnly): ExternalBillingAccount kind (eg aws)
+* **name**: string (Required, DeployTimeConstant): The resource name
+* **properties**: [ExternalBillingAccountProperties](#externalbillingaccountproperties) (ReadOnly): ExternalBillingAccount properties
+* **type**: 'Microsoft.CostManagement/externalBillingAccounts' (ReadOnly, DeployTimeConstant): The resource type
+
 ## Resource Microsoft.CostManagement/externalSubscriptions@2019-03-01-preview
-* **Valid Scope(s)**: ManagementGroup
+* **Valid Scope(s)**: Tenant, ManagementGroup
```

Comment:
I am curious as to *why* some of these resources are readonly at the tenant level. Is this an ARM runtime limitation? (I see that [the list of resources supported in tenant-scoped deployments](https://docs.microsoft.com/en-us/azure/azure-resource-manager/bicep/deploy-to-tenant?tabs=azure-cli#supported-resources) is very short, though I assumed that list was descriptive rather than normative.)